packages/config/config/devices/0x019b/tf016.json

```diff
@@ -96,6 +113,8 @@
 			"#": "3",
 			"$if": "firmwareVersion >= 1.8",
 			"label": "Floor sensor type",
+			"description": "The resistance (type) of the external room sensor.",
```

Comment:
Which is it? Floor sensor or external room sensor? Also I'm leaning towards unnecessary description.